apps/src/templates/projects/FeaturedProjectsTable.story.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import FeaturedProjectsTable from './FeaturedProjectsTable';
+
+// TODO [ErinB] update to use projectList data that highlights the differences of the tables. Currently a work in progress.
+export default storybook => {
+  storybook
```

Comment:
Tiny nit: You don't need the block part of this arrow function because the body consists of a single expression.

```js
export default storybook => storybook
  .storiesOf('FeaturedProjectsTable', module)
  .addStoryTable([
    // ...
  ]);
```